.golangci.yml

```diff
@@ -74,6 +74,7 @@ linters:
     - errcheck
     - exportloopref
     - funlen
+    - gocheckdirectives
```

Comment:
I added to the linter to the enabled list for golangci-lint, because it appears there are three cases where a go directive comment is being used incorrectly in this project and the linter detects them, so it seems like there might be some value in running this linter. 

```
/golangci-lint/test/ruleguard/dup.go:1:1: go directive contains leading space: // go:build
/golangci-lint/test/ruleguard/rangeExprCopy.go:1:1: go directive contains leading space: // go:build
/golangci-lint/test/ruleguard/strings_simplify.go:1:1: go directive contains leading space: // go:build
```

I can remove this line though if instructed.

Reply:
Actually, I removed the linter from here in 6c91740, and I can follow up with a separate PR about this to keep this PR hyper-focused on just adding the linter.